### PR TITLE
added renamed component prop

### PR DESCRIPTION
**Pull request by dbilgin**
Added [128](https://github.com/KyleAMathews/react-headroom/issues/128)

### CODE REVIEW

**Comment by janczizikow** [comment]
Hey @dbilgin thanks for the PR! Couple things:

1. could you destructure the `tag` prop above? I think the props are already destructured at the very beginning of the render method.
2. you should delete the `tag` prop from `divProps`, because right now it will be applied to the inner div in `{...rest}`. 
3. can you add some documentation for it in `www/pages/index.md`?

**Comment by dbilgin** [comment]
@janczizikow done! could you check again?

File: www/pages/index.md [hunk 1]
```diff
@@ -76,3 +76,4 @@ Another option is to use CSS. The component has a `headroom` class as well as a
 *   `wrapperStyle` — pass styles for the wrapper div (this maintains the components vertical space at the top of the page).
 *   `parent` — provide a custom 'parent' element for scroll events. `parent` should be a function which resolves to the desired element.
 *   `pinStart` — height in px where the header should start and stop pinning. Useful when you have another element above Headroom component.
+*   `tag` - change the wrapper with the tag of your choice.
```
Comment (janczizikow) [inline www/pages/index.md:79]:
```suggestion
*   `tag` - change the wrapper html element type. Default - `"div"`.
```

**Review by janczizikow — APPROVED** [review]
Thanks, looks good to me!

**Pull request merged**